hypha/apply/funds/views.py

```diff
@@ -1157,3 +1162,33 @@ def get_submission_values(self):
                 values.append(value)
 
         return {'total': sum(values), 'average': round(mean(values))}
+
+
+@method_decorator(login_required, name='dispatch')
+class ReviewLeaderboard(SingleTableMixin, FilterView):
+    filterset_class = LeaderboardFilter
+    table_class = LeaderboardTable
+    table_pagination = False
+    template_name = 'funds/review_leaderboard.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        is_staff = request.user.is_apply_staff
+        is_reviewer = request.user.is_reviewer
+
+        if not (is_staff or is_reviewer):
+            raise PermissionDenied
+
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_table_data(self):
+        ninety_days_ago = timezone.now() - timedelta(days=90)
+        this_year = timezone.now().year
+        last_year = timezone.now().year - 1
+        # latest_reviews = Review.objects.filter(author__reviewer_id=OuterRef('pk')).order_by('-created_at')
```

Comment:
The comment out parts make this far too slow with live data. Better way to do this?

Could we base the query on the Review model instead and calculate review per user from that data?

Reply:
Looks like the cause of slowness is in the `render_most_recent ` method. Adding a comment there.